src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs

```diff
@@ -0,0 +1,74 @@
+using FluentAssertions;
+using Stripe.Tests.Xunit;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Stripe.Tests.XUnit
+{
+
+
+    public class when_listing_issuer_fraud_records
```

Comment:
Can you make sure to just tighten up the spacing in these for the final version?

Reply:
Oh I copied the spacing from when_listing_invoices, sorry I thought that was convention. Will remove spaces